nebula_ros/src/hesai/hw_monitor_wrapper.cpp

```diff
@@ -227,9 +227,8 @@ void HesaiHwMonitorWrapper::HesaiCheckStatus(diagnostic_updater::DiagnosticStatu
   std::scoped_lock lock(mtx_lidar_status_);
   if (current_status_) {
     json data = current_status_->to_json();
-    for (const auto & [key, value] : data.items()) {
-      diagnostics.add(key, value);
-      std::cout << key << " : " << value << std::endl;
+    if(data.contains("motor_speed")) {
```

Comment:
In `HesaiCheckStatus`, everything **but** `"motor_speed"` and `"temperature"` should be checked, please add the for-loop back in here and do
```cpp
if (key == "motor_speed" || key == "temperature") continue;
```